filebeat/registrar/registrar.go

```diff
@@ -27,6 +27,7 @@ type Registrar struct {
 
 var (
 	statesUpdated = expvar.NewInt("registrar.state_updates")
+	statesTotal   = expvar.NewInt("registar.states.total")
```

Comment:
@urso let me add more counters in a separate PR. I added a note to https://github.com/elastic/beats/issues/1913